sprout/core/write_file.py

```diff
@@ -0,0 +1,16 @@
+from pathlib import Path
+
+
+def write_file(path: Path, string: str) -> Path:
+    """Writes a file to the given path with the given content.
+
+    Args:
+        path: Path of the file to be created, including file name and extension.
+        string: Content to be written to the file.
```

Comment:
```suggestion
        string: Content to be written to the file.
        path: Path of the file to be created, including file name and extension.
```

Just to match the order.